### ISSUE TITLE
There is a content collision between a button category and the outside of the button

### ISSUE THREAD

**Issue by ollie-iterators** [CityOfBoston/Iterators#133]
Screengrab/videos (insert file names here)
2020 08 19, Content Collision on article button on main page.mp4
[2020 08 19, Content Collision on article button on main page.mp4.zip](https://github.com/CityOfBoston/Iterators/files/5099050/2020.08.19.Content.Collision.on.article.button.on.main.page.mp4.zip)

Link
https://d8-stg.boston.gov/

Node # (if known)
21

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer > MacOS 10.15.6

Trusted Tester Test

Brief Description
There is a content collision between a button category and the outside of the button

Steps to reproduce
1)	Go down to the Latest News section on the main page
2)	00:00 Check out the article for ‘Aug 17’

Desired Result
•	All of the content in the button is contained in the button

Actual Result
•	The category of the button collides with the edges of the button

**Comment by subaha-cob** [comment]
@andrebtlima for content collision tickets we need to chat about them.

**Comment by subaha-cob** [comment]
I reduced the size of the content. I hope the title still makes sense.
Test on: https://d8-uat.boston.gov/

**Comment by andrebtlima** [comment]
@subaha-cob I think this is fixed cosmetically for the homepage, but yes I agree this should be a larger conversation

**Comment by jill-iterators** [comment]
The content is now different on the UAT and we are unable to verify this particular news story from August 17. Close the file, continue to monitor if we see this again.
